registry/config_windows.go

```diff
@@ -6,10 +6,10 @@ import (
 	"strings"
 )
 
-// CertsDir is the directory where certificates are stored
-func CertsDir() string {
-	return os.Getenv("programdata") + `\docker\certs.d`
-}
+// defaultCertsDir is the platform-specific default directory where certificates
+// are stored. On Linux, it may be overridden through certsDir, for example, when
+// running in rootless mode.
+var defaultCertsDir = os.Getenv("programdata") + `\docker\certs.d`
```

Comment:
```suggestion
const defaultCertsDir = os.Getenv("programdata") + `\docker\certs.d`
```

Reply:
Hehe, I tried, but that doesn't work because of the `os.Getenv()` (so I decided to use what we could get, and used a `const` for linux, and kept it as a var for Windows)

```
 os.Getenv("programdata") + `\docker\certs.d` (value of type string) is not constant
```